Gifski/CropOverlayView.swift

```diff
@@ -0,0 +1,827 @@
+//
+//  CropScreen.swift
+//  Gifski
+//
+//  Created by Michael Mulet on 3/23/25.
+
+import SwiftUI
+import AVFoundation
+import AVKit
+
+fileprivate let cornerWidthHeight = 20.0
+
```

Comment:
The logic in this file is not very maintainable. Would be nice to have it cleaned up, simplified. There is a lot of repetition that could be DRY'd up. Should probably happen when the logic is finalized though.